apps/deploy_monitoring/src/constants/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Port constants
```

Comment:
@adonagy 
missing 
```
// Copyright (c) SimpleStaking and Tezedge Contributors
// SPDX-License-Identifier: MIT
```

and I would just create constants.rs insetad of subdir constants/mod.rs